web/src/lib/components/asset-viewer/activity-viewer.svelte

```diff
@@ -87,11 +85,16 @@
 
   onMount(async () => {
     await getReactions();
+    activityManager.init(albumId, assetId);
+  });
+
+  onDestroy(() => {
+    activityManager.reset();
   });
 
   const getReactions = async () => {
     try {
-      reactions = await getActivities({ assetId, albumId });
+      await activityManager.updateActivities(albumId, assetId);
```

Comment:
`updateActivities` seems weird. You aren't doing an http update. Maybe it should be something like `refresh`